android-components/components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/SentryService.kt

```diff
@@ -136,6 +137,9 @@ class SentryService(
             options.environment = environment
             options.beforeSend = BeforeSendCallback { event, _ ->
                 val throwable = event.throwable
+                if (event.level == SentryLevel.FATAL) {
+                    alterEventToBeUnhandled(event)
+                }
```

Comment:
❓ Should we add some tests for this?

Reply:
Yes, definitely! I pulled out the `BeforeSendCallback` and wrote some tests for it :)